frontend/src/features/acquisition/list/AcquisitionSearchResults/AquisitionProperties.tsx

```diff
@@ -0,0 +1,71 @@
+import { LinkButton } from 'components/common/buttons';
+import { Api_AcquisitionFileProperty } from 'models/api/AcquisitionFile';
+import { useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { formatApiAddress } from 'utils';
+
+export interface IAcquisitionPropertiesProps {
+  aquisitionProperties?: Api_AcquisitionFileProperty[];
+  maxDisplayCount: number;
+}
+
+const AcquisitionProperties: React.FunctionComponent<IAcquisitionPropertiesProps> = props => {
```

Comment:
This seems like a good candidate for a common component given that LeaseProperties.tsx is nearly identical except for the property list (although both lists share pid, pin, address. What was your thinking on keeping the two components separate?

Reply:
I'd be more ok with duplicating this component if we had test coverage on the original. Since we don't we are losing some test coverage % by duplicating here.

Reply:
That is a good observation, I thought it too. For now, I believe it would be better to keep the duplicated component and in the future remove it if there are no additional changes needed.
The problem with refactoring now is that I am not certain that the business wants/will want the same information on both sides